js-intro.md

```diff
@@ -5,68 +5,73 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is a function? Why would you use one?
 
-  Your answer:
+  Your answer: a function is a declared series of code that can be called to carry out a certain action.
+  I would use it to quickly call a series of code without having to write out the code line by line.
 
-  Researched answer:
 
+  Researched answer: a function is "a named section of a program that performs a specific task". "Function are used for divide a large code into module, due to this we can easily debug and maintain the code."
 
 
-2. What is the difference between map and filter?
 
-  Your answer:
+2. What is the difference between map and filter?
 
-  Researched answer:
+  Your answer: .map goes through an array and commits a designated action to each element, spitting out a new array with the altered values. .filter goes through an array and makes a new array with only the values that meet a certain given condition.
 
+  Researched answer: ". filter() returns a subset of the elements from the original array, while . map() produces an array with new, different entries based on the elements in the original array."
 
 
 3. What is the difference between console.log() and return?
 
-  Your answer:
+  Your answer: console log logs the value on to the computer without retaining the value whereas return holds the value while not logging it.
 
-  Researched answer:
+  Researched answer: "console. log() is a function used to print information to the console. return on the other hand is a call to pass some value back up to where the call was made"
 
 
 
 4. In regards to functions, what is an argument?
 
-  Your answer:
+  Your answer: an argument is the value one puts in into a function.
```

Comment:
Just to add to this, we know that arguments are specifically outside information that some functions require in order to execute code correctly and accurately.